diff --git a/config/meson.build b/config/meson.build
index 518ddf1..0ea7f5a 100644
--- a/config/meson.build
+++ b/config/meson.build
@@ -266,14 +266,7 @@ endif
 ## ========================================= ##
 ## populate the data for crest_metadata.fh
 ## ========================================= ##  
-commit = 'unknown commit'
-git = find_program('git', required: false)
-if git.found()
-  git_commit = run_command(git, 'show', '-s', '--format=%h',check:true)
-  if git_commit.returncode() == 0
-    commit = git_commit.stdout().strip()
-  endif
-endif
+commit = 'v3.0.2'
 # create configuration data
 config = configuration_data({
   'name': meson.project_name(),
@@ -281,8 +274,8 @@ config = configuration_data({
   'description': 'Conformer Rotamer Ensemble Sampling Tool',
   'commit': commit,
   'date': run_command('date',check:true).stdout().     strip(),
-  'author': run_command('id','-u','-n', check:true).stdout().strip(),
-  'origin': run_command('hostname', check:true).stdout().strip(),
+  'author': 'conda-forge',
+  'origin': 'conda-forge',
   'fcid': fc.get_id(),
   'fcver': fc.version(),
   'ccid': cc.get_id(),
